<launch>
  <node pkg="miradar_node" name="miradar_node" type="miradar_node" output="screen">
    <param name="devicename" value="/dev/ttyACM0" />
  </node>
  <node pkg="miradar_node" name="ppi_visualizer" type="ppi_visualizer.py" />
  <node pkg="tf" name="tf_publisher" type="static_transform_publisher" args="0 0 0 0 0 0 map miradar 10" />
  <node pkg="tf" name="tf_publisher_scan" type="static_transform_publisher" args="0 0 0 0 0 0 miradar miradar_scan 10" />
</launch>
